### PR TITLE
Smash feature

### PR DESCRIPTION
**Pull request by mcihadkurel**
### Smash Magazine clone
**This project made by full of grid and flexbox**
1-) Fixed all validator errors
2-) Given an absolute value for "h" tag top-middle of page.
3-) Fixed all linter errors.
4-) Added embedded images in sections.
5-) Used relative styling in CSS.
6-) Changed background colors as grayscale.
7-) Removed artıcle wıdths.
8-) Removed body's background color.

### CODE REVIEW

**Comment by bertil291utn** [comment]
# Status: on review 🟠 

##### *Reach out to me on* [Twitter](bit.ly/bertil-twitter) , [LinkedIn](bit.ly/bertil_linkedin) or Slack Microverse-community as @btandayamo

File: index.html [hunk 1]
```diff
@@ -24,9 +24,79 @@
...
+            </div>
+            <div class="artfooter">
+                <i class="fa fa-long-arrow-right" aria-hidden="true"></i>
+            </div>
+         </div>
+         <div class="rightup">
+            <div class="writer">
+                <img src="./images/timi-oyeni-200x200.jpg" alt="writer">
+                <p>Chidi orji</p><span>wrote</span>
+            </div>
```
Comment (bertil291utn) [inline index.html]: Use `<article>` tag instead div ones (classes left up, right up and others), and instead `<article>` tag use `<section>` one. As to approach like an actual page

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🔴  

# Hey guys, you've been doing a great job and also I hope you enjoyed writing.
But in order to accomplish as the actual page, you required to do these changes, listed below: 

#### Article tags

- [ ] Use article tags to write each article _(I left an inline comment )_

image


#### Float porperty

image


- [ ] Also you're forgetting work with float property
Take advantage of working with these elements

image


#### Horizontal scrolling 
- [ ]  To avoid horizontal scrolling try adding overflow-x: hidden on your body element

**Your design** 

image

#### 2 articles rows

- [ ] It contains 2 or 3 articles, each takes half or third of the width respectively  (content is not important)
 When you have 2, 3 o more articles you have to be careful with the width, try approach as the actual page 

**Yours**

image

**Actual smashing page**

_With 2 articles_

image
 
_With 3 articles_

image


##### Footer

* [ ] It contains centered links (content is not important) ([link to page](https://www.smashingmagazine.com/))

image

[TAKE IN MIND]
Your commits messages are meaningful but you could improve writing more descriptive ones, some are descriptive others are not.
You might try this template:
```
Title (non-descriptive)
- Explaining with more detailed words
- Another detailed feature same commit

```
 
Guys do the changes requested above and request another review.
Good luck. 
:shipit: ? Not yet
 
> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

##### Do you have any question? *Reach out to me on* [Twitter](bit.ly/bertil-twitter), [LinkedIn](bit.ly/bertil_linkedin) or Slack Microverse-community like @btandayamo

**Comment by mcihadkurel** [comment]
> ### CHANGES REQUIRED ♻️ 🛑
> ### HELLO @mcihadkurel AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT.
> ### I LIKE YOUR PROJECT ALOT 👏 ACTUALLY I WOULD HAVE APPROVED IT BUT YOU NEED TO WORK ON THE FOLLOWING ⬇️
> * [ ]  There are still some errors in your linter. You need to fix all of them before the project can be approved.
> 
> **Your project is very impressive good work on making the requested changes by the previous TSE**
> 
> If you have any question you can reach me via slack on @peterrobert
> 
> ### HAPPY CODING 💻

**Pull request merged**